ui/src/shared/components/FluxMonacoEditor.tsx

```diff
@@ -0,0 +1,35 @@
+// Libraries
+import React, {FC} from 'react'
+
+// Components
+import MonacoEditor from 'react-monaco-editor'
+import {tokenizeFlux} from 'src/external/monaco.fluxLang'
+import {addFluxTheme} from 'src/external/monaco.fluxTheme'
+import {addSnippets} from 'src/external/monaco.fluxCompletions'
+
+interface Props {
+  script: string
+}
+
+const FluxEditorMonaco: FC<Props> = ({script}) => {
+  const editorWillMount = monaco => {
```

Comment:
this is pretty cool way to get the instance, is this a different library than the one we were playing with?